algorithms/2-search-solutions/binary_search_iterative.rb

```diff
@@ -0,0 +1,17 @@
+def binary_search(sorted_array, value)
+  collection_size = sorted_array.length - 1
```

Comment:
That is not a good name for that variable, since it is actually size minus one. You should just put `high` here instead.